### DISCUSSION TITLE
Is there a way to rename a Dagster project?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17632] [Q&A]
Hi everyone, is there a way to rename a dagster project? Thanks!

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1698922024753379?thread_ts=1698922024.753379&cid=C01U954MEER).*

**Comment by sryza** [comment]
Are you able to add some more detail about what exactly you want to rename? "Project" is not an explicit concept in Dagster.

E.g. code location? Python package that your Dagster code lives in? Folder that your Dagster code lives in?
